modules/light-clients/14-avalanche/client_state_test.go

```diff
@@ -194,3 +210,876 @@ func (suite *AvalancheTestSuite) TestInitialize() {
 		}
 	}
 }
+
```

Comment:
Some of the tests in the tendermint client for `TestVerifyMembership` aren't here, like "successful ConsensusState verification", "successful Connection verification", etc. Do we need those?